shell/app-shell/lib/arcs-utils.js

```diff
@@ -40,8 +40,9 @@ const ArcsUtils = {
   },
   createUrlMap(cdnRoot) {
     // Module import not available in workers yet, we have to use the build for now
-    //const lib = document.URL.includes('debug') ? 'source' : 'build';
-    const lib = 'build';
```

Comment:
Did the changes to `arcs-util` sneak in from a different cl? I don't see any other wiring for the transpiler in particular.

Reply:
Yes, this isn't meant to be here :stuck_out_tongue: I'll remove it before merging.